ported/libibverbs/rc_pingpong.c

```diff
@@ -639,12 +639,16 @@ int main(int argc, char *argv[])
 		} else {
 			do {
 				rd = fi_cq_read(ctx->cq, &wc, 1);
+				if (rd == -FI_EAGAIN) {
+					rd = 0;
+					continue;
+				}
 			} while (rd == 0);
```

Comment:
It would be cleaner to change the while loop to check for:
while (rd == -FI_EAGAIN)